src/olympia/amo/management/commands/fix_licenses.py

```diff
@@ -0,0 +1,68 @@
+from django.core.management.base import BaseCommand
+from django.utils.translation import activate
+
+from olympia.versions.models import License
+from olympia.addons.models import Addon
+from olympia.devhub.models import ActivityLog
+from olympia.translations.models import Translation
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        activate('en-us')
+
+        mpl11 = License.objects.get(pk=5)
+
+        assert str(mpl11.name) == 'Copyright Jason Savard', 'wrong license'
+
+        # set on_form explicitly to False to make sure it's not
+        # visible and set `builtin` to 1 to make sure it's filtered
+        # everywhere.
+        mpl11.on_form = False
+        mpl11.builtin = 1
+        mpl11.save()
+
+        search = '"versions.license": 5}'
+        changed_addons = ActivityLog.objects.filter(
+            action=37,
+            _arguments__contains=search)
+
+        msg = 'too many or few add-ons changed, found %s' % len(changed_addons)
+        assert len(changed_addons) == 3, msg
+
+        for log in changed_addons:
+            license, addon = log.arguments
+
+            assert isinstance(license, License)
+            assert isinstance(addon, Addon)
+
+            last_version = addon.get_version()
+
+            # Create new object of broken license since they were supposed
+            # to change and show them in the list again
+            last_version.license.pk = None
+            last_version.license.on_form = True
```

Comment:
oh, haha, yeah, the `on_form` is wrong. Fixing this with the other stuff, but I just tested, setting `pk = None` makes django create a new instance. But it's actually easier to create a completely new one and copy over the data because of how the translations are bound :-)